test/PrefilterExpressionIndexTest.cpp

```diff
@@ -112,6 +128,17 @@ class PrefilterExpressionOnMetadataTest : public ::testing::Test {
   const LocalVocabEntry iri5 = LVE("<www-iri.de>");
   const LocalVocabEntry iriBegin = LVE("<");
   const Id idAugsburg = getId(augsburg, vocab);
+  const Id vocabIdBe = getVocabId("\"Be\"");
+  const Id vocabIdBern = getVocabId("\"Bern\"");
+  const Id vocabIdBerlin = getVocabId("\"Berlin\"");
+  const Id vocabIdDüsseldorf = getVocabId("\"Düsseldorf\"");
+  const Id vocabIdH = getVocabId("\"H\"");
+  const Id vocabIdHam = getVocabId("\"Ham\"");
+  const Id vocabIdHamb = getVocabId("\"Hamb\"");
+  const Id vocabIdHamburg = getVocabId("\"Hamburg\"");
+  const Id vocabIdHamburgAltona = getVocabId("\"Hamburg Altona\"");
+  const Id vocabIdMünchen = getVocabId("\"München\"");
+  const Id vocabIdStuttgart = getVocabId("\"Stuttgart\"");
```

Comment:
You should make some of them `LocalVocabEntry` (at the correct position, so the same position)
(Some LVEs with entries that are also in the vocabulary, and some that are not. 
Then make sure to cover all possible cases.